config/config.ini

```diff
@@ -2043,6 +2043,9 @@ name = Shannon -jj Behrens
 [http://shiningpanda.com/feeds/all.atom.xml]
 name = ShiningPanda
 
+[https://tech.shopkick.com/tagged/python/atom.xml]
+name = Shopkick Tech Blog
```

Comment:
Please sort this by URL, rather than by name.  The fastest way to get this right is in a clone of your fork:

```bash
$ cd config
$ python sort-ini.py
$ git commt --amend
$ git push -f mahmoud:patch-1
```